src/ui/settings/pages/AssetBrowser.tsx

```diff
@@ -11,18 +11,14 @@ export default function AssetBrowser() {
     return (
         <ErrorBoundary>
             <RN.View style={{ flex: 1 }}>
-                <Search
-                    style={{ margin: 10 }}
-                    onChangeText={(v: string) => setSearch(v)}
-                    placeholder="Search"
-                />
+                <Search style={{ margin: 10 }} onChangeText={(v: string) => setSearch(v[0] === "*" ? v : v.toLowerCase())} placeholder="Search" />
                 <RN.FlatList
-                    data={Object.values(all).filter(a => a.name.includes(search) || a.id.toString() === search)}
+                    data={Object.values(all).filter((a) => (search[0] === "*" ? a.name.includes(search.slice(1)) : a.name.toLowerCase().includes(search)) || a.id.toString() === search)}
```

Comment:
In what case would case-sensitive searching be needed?

Reply:
for the new tabs v2 icons

Reply:
Isn't it perfectly fine to just also search them case-insensitive?